src/mvMetricsWindow.cpp

```diff
@@ -74,16 +75,16 @@ void mvMetricsWindow::drawWidgets()
             ImGui::Text("Application average %.3f ms/frame (%.1f FPS)", 1000.0f / io.Framerate, io.Framerate);
             ImGui::Text("%d vertices, %d indices (%d triangles)", io.MetricsRenderVertices, io.MetricsRenderIndices, io.MetricsRenderIndices / 3);
             ImGui::Text("%d active windows (%d visible)", io.MetricsActiveWindows, io.MetricsRenderWindows);
-            ImGui::Text("%d active allocations", io.MetricsActiveAllocations);
 
             static std::map<std::string, ScrollingBuffer> buffers;
             static float t = 0;
             t += ImGui::GetIO().DeltaTime;
 
             const auto& results = mvInstrumentor::Get().getResults();
 
-            for (const auto& item : results)
+            for (const auto& item : results) {
                 buffers[item.first].AddPoint(t, (float)item.second.count());
+            }
```

Comment:
Adding curly braces for the sake of adding curly braces, and in a different style... I'd say let's roll back these two lines.